pkg/destroy/bootstrap/bootstrap.go

```diff
@@ -62,11 +69,14 @@ func Destroy(dir string) (err error) {
 		}
 	}
 
-	if platform == libvirt.Name {
+	if !destroy || platform == libvirt.Name {
```

Comment:
both libvirt and aws are going to call apply to `disablle-bootstrap` so this `platform == libvirt.Name` is redundant?

Reply:
just copying the logic we had here before. libvirt had this additional apply run already for some reason. Keep in mind that via the destroy command (which shouldn't and probably can't be graceful) this is called only in destroy mode. With my change here I keep the old libvirt logic in that case.